.github/workflows/test-and-publish-nightly.yml

```diff
@@ -43,6 +47,9 @@ jobs:
       image: ${{ matrix.image }}
     name: "Build and unit test (${{ matrix.runson.display }})"
     steps:
+      - name: Update PATH
+        if: matrix.runson.display == 'amazonlinux2'
+        run: echo "/usr/local/bin" >> $GITHUB_PATH
```

Comment:
Can paths be configured in the image? (or tools be installed on a path accessible by default?)
The idea is to minimize image-specific configuration as much as possible.